src/nvim/screen.c

```diff
@@ -7810,20 +7826,20 @@ void limit_screen_size(void)
 
 void win_new_shellsize(void)
 {
-  static long old_Rows = 0;
```

Comment:
These matches the `Row` and `Columns` globals, so these shouldn't  be changed unless these are changed as well (which would be fine, if you want to do it).

Reply:
> unless these are changed as well (which would be fine, if you want to do it).

could cause a lot of shadowing, i would suggest renaming the globals to `g_rows`, `g_columns` if we are going to rename them.